src/transformStateWithClones.js

```diff
@@ -66,6 +66,29 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const copyState = { ...state };
```

Comment:
copyState is more relevant name for a function, but for a variables we use nouns

```suggestion
  const stateCopy = { ...state };
```